src/main/resources/db/migration/V202408091410__owner_table.sql

```diff
@@ -0,0 +1,9 @@
+create table owner(
+    owner_id serial primary key ,
```

Comment:
rename to id, the owner part is redundant in the context of owner

Reply:
got it!